obs-packaging/create-repo-branch.sh

```diff
@@ -47,7 +47,7 @@ read_repos(){
 		[[ "$p" != "#"* ]] || continue
 		repos+=("${p}")
 		echo "Adding distro: ${p}"
-	done < "${script_dir}/distros"
+	done < "${script_dir}/distros_${arch_target}"
```

Comment:
I like the idea, but please rename the current `distros` file to `distros_x86_64`